docs/quick-starts/your-first-experiment-analysis.md

```diff
@@ -41,7 +41,7 @@ This quickstart assumes that you already have assignment logs tracked in your da
 
 If you do not have a experiment assignment tool integrated, please refer to the [experiment assignment](../experiments/prerequisites/experiment-assignment) section for more instructions.
```

Comment:
While you are at it, I think it would be nice to move this line above the table (before the In particular, Eppo requires.... sentence)

Reply:
awesome, jotted this down for when I update the quickstarts later this week